InternshipBE/DAL/Validators/EvaluationValidator.cs

```diff
@@ -0,0 +1,9 @@
+﻿using DAL.Entities;
+
+namespace DAL.Validators
+{
+    public class EvaluationValidator : Validator<Evaluation>
+    {
+
```

Comment:
unnecessary space

Reply:
It is git illusion. ✨
Unnecessary space is not exist in real project. 🙃